### PR TITLE
Add matrix item and update copy

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated landing page as per [doc](https://docs.google.com/document/d/1mcfPYByoIT_iFOot6fxqq_bXX9p9vENWkLt4-ZI22Qg/edit)

## QA

- Visit demo https://docs-ubuntu-com-358.demos.haus/
- See that requested changes were made 

## Issue / Card

https://warthogs.atlassian.net/browse/WD-4994

## Screenshots

image

### CODE REVIEW

File: static/sass/main.scss [hunk 1]
```diff
@@ -14,3 +14,9 @@ td {
 is-dark {
   background-color: $theme-default-nav;
 }
+
+.is-last {
```
Comment (anthonydillon) [inline static/sass/main.scss:18]: This is fine but might be improved with the selector that works forever:
```css
li.p-matrix__item:last-of-type:nth-of-type(3n + 1) {
```
Maybe propose this to Vanilla?

**Review by anthonydillon — APPROVED** [review]
LGTM :+1: with a suggestion in the code

File: static/sass/main.scss (hunk 1 above)
Comment (mtruj013) [reply static/sass/main.scss]: Proposed here https://github.com/canonical/vanilla-framework/issues/4828

**Comment by lyubomir-popov** [comment]
I would add a list style like so (somethjing we can componentise in the [lists roadmap item](https://warthogs.atlassian.net/browse/WD-7133?atlOrigin=eyJpIjoiOTUyMjU0MDExNmM1NDRjOTk3MWJmMTMwYjg3ZTY3N2QiLCJwIjoiaiJ9)):
image

**Pull request merged**